stock_orderpoint_automatic_creation/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Stock Orderpoint Automatic Creation
```

Comment:
Update to latest README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst